CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+
+#### OTTR paper
+
+There's an [OTTR paper](https://pubmed.ncbi.nlm.nih.gov/37207236/) published in the Journal of Statistics and Data Science Education by C. Savonen, C. Wright, A. Hoffman, et al. in 2023. [For information on citing the manuscript, see here](https://www.ottrproject.org/index.html#how-to-cite-ottr)
+
+#### Cheatsheets
+
+TBA
+
+#### Videos
+
+TBA
+
+## How should you suggest changes or alert us about problems?
+
+There are various ways in which you can request or suggest changes to OTTR. Below we discuss ways in which you can make suggestions or requests, but leave the implementation to others or implement the suggestions yourselves.
+
+### You want to make suggestions, but leave the implementation to someone else
+
+If you want to alert us about an issue or make suggestions, you can open a GitHub Issue or submit a response in our OTTR Feedback Google Form.
+
+#### GitHub Issues
+
+Open an issue describing the bug or feature request for the relevant repository using the following links.
+
+| Repository            | Issues Link                                                                 |
+|:---------------------:|:---------------------------------------------------------------------------:|
+| OTTR_Template         | [Open an issue here](https://github.com/jhudsl/OTTR_Template/issues/new/choose)        |
+| OTTR_Quizzes          | [Open an issue here](https://github.com/jhudsl/OTTR_Quizzes/issues/new/choose)         |
+| OTTR_Template_Website | [Open an issue here](https://github.com/jhudsl/OTTR_Template_Website/issues/new/choose)|
+| ottr-reports          | [Open an issue here](https://github.com/jhudsl/ottr-reports/issues/new/choose)         |
+| ottrpal               | [Open an issue here](https://github.com/jhudsl/ottrpal/issues/new/choose)              |
+| cow                   | [Open an issue here](https://github.com/jhudsl/cow/issues/new/choose)                  |
+| ottrproject.org       | [Open an issue here](https://github.com/jhudsl/ottrpal/issues/new/choose)              |
+
+##### Description of choices for issue type (if given)
+
+Most of the links above will directly open an issue for you to fill out a description of the problem or request, but several of the repositories provide choices (and related description templates) for the type of issue you want to open. Those choices could include the following:  
+
+* **Feature Request** or **Course Template Feature Request**: Use this issue if you want to make a suggestion for an addition.
+* **Problem Report** or **Course Template Problem Report**: Use this issue if you want to report a problem or possible improvement.
+* **New Content Idea**: This is a template meant for opening an issue in an OTTR course, so that someone can suggest an additional topic or idea for that specific course. It shouldn't be used for opening an issue in one of these repos.
```

Comment:
```suggestion
* **New Content Idea**: This is a template meant for opening an issue in an OTTR course, so that someone can suggest an additional topic or idea for that specific course. It can be used for general suggestions for content. 
```